### DISCUSSION TITLE
Zoom in on individual features when acquisition is paused

### DISCUSSION THREAD

**Discussion by fultond** [open-ephys/plugin-GUI#531] [Ideas]
Dear Open-Ephys team,
Good to meet you at SFN. We had a chat about the GUI and discussed that it would be useful to be able to be able to zoom in on individual features once acquisition is paused - this function would be useful for a quick exploration of waveforms, e.g. to discriminate non-biological signals (square pulses) from biological (waveforms with kinetics expected of action potentials / EPSPs ...). So in other words, to be able to change the timebase on the paused acquisition window in order to zoom in on fast signals. 
This could be a handy feature in the GUI.
Hope this makes sense! 
BW
Daniel

**Comment by jsiegle** [comment]
Hi Daniel – we agree this feature would be super useful. It will take some effort to get this to work, though. I will leave this issue open and send an update once we have a prototype implemented.

**Reply by fultond** [reply]
That sounds good, thanks for looking at this and glad the suggestion makes sense. 
Daniel
